components/ledger/cmd/serve.go

```diff
@@ -30,8 +30,15 @@ const (
 func NewServe() *cobra.Command {
 	cmd := &cobra.Command{
 		Use: "serve",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			if err := bindFlagsToViper(cmd); err != nil {
+				return err
+			}
+
+			return nil
+		},
```

Comment:
Maybe use `cobra.EnableTraverseRunHooks = true`
It allow to traverse All PersitentPreRun From Parent to Childs Nodes

Reply:
We still have specific flags for commands, so we have to bind them for every command we have

Reply:
Let's do that in another PR, since it means migrating from the base cobra repository to our fork. This PR is already big enough !